docs/team/lfrostbytee.md

```diff
@@ -5,28 +5,43 @@ title: Ong Wee Marcus's Project Portfolio Page
 
 ### Project: Duke The Market
 
-Duke The Market is a desktop contact management system application 
-used for managing customer contacts in department stores. The user interacts with it using a CLI, 
-and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Duke The Market is a desktop application used for managing customer contacts and events in department stores. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. 
+It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **Code contributed**: to be added soon
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=lfrostbytee&breakdown=true)
 
 * **Enhancements implemented**:
-  * to be added soon
+    * Updated the GUI for the application (Pull requests [\#54](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/54), [\#131](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/131))
```

Comment:
Perhaps can specify what the update managed to accomplish!
```suggestion
    * Updated the GUI for the application to display `event`(s) (Pull requests [\#54](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/54), [\#131](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/131))
```

Reply:
Actually is there a reason you didn't want the What it does & Justification section for this enhancement?